### PR TITLE
Added Frontend Routes and renamed all "Register" items to "SignUp"

### LINKED ISSUE #13: Setup frontend routes
## Instructions
The following are routes our app currently needs:
- Login → `/login`
- Sign up → `/signUp`
- Logout → `/logout`
- Forgot Password → `/forgotPassword`
- AdminDashboard (protected) → `/dashboard`
- Catalog (protected) → `/catalog`
  - create this page in `src/pages/Catalog/Catalog.jsx`
- Published Schedule (protected) → `/publishedSchedule`
  - create this page in `src/pages/PublishedSchedule/PublishedSchedule.jsx`
- Playground → `/playground`
  - This where people can test their components

Add the routes and associated pages (create empty page with page name as a header if it doesn’t exist yet) to the App.jsx file

## Notes
- Make sure that pages with protected access can only be accessed once logged in
- All the auth pages should already be created

## Acceptance Criteria
- All the specified routes load and protected routes only work if logged in

## Resources
- [Route Setup Example](https://github.com/ctc-uci/patriots-and-paws-frontend/blob/dev/src/App.jsx#L116)
- [React Router Docs](https://reactrouter.com/en/main/start/tutorial)

### PR DESCRIPTION
**Pull request by h0ethan04**
Added frontend routes to App.jsx; created new pages for Playground, Published Schedule and Catalog. 
Changed login path to` '/login'` from `'/'` and renamed all items and files from "Register" to "SignUp" to better fit design plans.
closes #13

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -37,6 +40,26 @@ const App = () => {
                   />
                 }
               />
+              <Route
+                exact path="/catalog"
+                element={
+                  <ProtectedRoute
+                    Component={Catalog}
+                    redirectPath="/"
```
Comment (michellelin1) [inline src/App.jsx]: could you change the redirectPath to `/login` for all the protected routes?

File: src/App.jsx [hunk 2]
```diff
@@ -21,10 +24,10 @@ const App = () => {
         <CookiesProvider>
           <Router>
             <Routes>
-              <Route exact path="/" element={<Login />} />
```
Comment (michellelin1) [inline src/App.jsx]: we still want the `/` path, could that load the `PublishedSchedule` page?

**Review by michellelin1 — DISMISSED** [review]
thanks for getting your PR in early! Everything looks good, just a few minor changes: change the redirect path to `/login` and add back the `/` route for PublishedSchedule page (you can still leave `/publishedSchedule` for now). Once those are addressed, this PR should be good :)

**Review by ThatMegamind — DISMISSED** [review]
Thanks for making those changes, looks good now!

**Review by ThatMegamind — APPROVED** [review]
Looks good to us!

**Pull request merged**